indico/htdocs/sass/base/_defaults.scss

```diff
@@ -90,3 +90,18 @@ $default-box-shadow-v-offset: 1px;
     background: $start-color;
     @include background-image(linear-gradient(top, $start-color, $end-color));
 }
+
+@mixin placeholder($color: inherit) {
```

Comment:
iirc. we said during the meeting that it should be called `placeholder-color` :)